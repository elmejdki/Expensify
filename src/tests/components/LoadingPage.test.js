import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import LoadingPage from '../../components/LoadingPage';

const renderer = new ShallowRenderer();

it('should render LoadingPage correctly', () => {
  renderer.render(<LoadingPage />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
