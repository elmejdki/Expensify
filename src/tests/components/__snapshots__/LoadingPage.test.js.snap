// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage correctly 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
